Mathlib/Algebra/Star/Basic.lean

```diff
@@ -124,18 +124,18 @@ attribute [simp] star_trivial
 /-- A `*`-semigroup is a semigroup `R` with an involutive operation `star`
```

Comment:
This docstring and the module docstring should be updated to reflect the new generality.

Reply:
Done.